contracts/ERC20SimpleSwap.sol

```diff
@@ -1,8 +1,11 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
If this is merged, we can close https://github.com/ethersphere/swap-swear-and-swindle/issues/122